packages/govuk-frontend/src/govuk/components/date-input/date-input.yaml

```
@@ -31,11 +31,11 @@ params:
       - name: autocomplete
         type: string
         required: false
-        description: Attribute to [identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html), for instance `"bday-day"`. See [autofill](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) for full list of attributes that can be used.
+        description: 'Attribute to meet [WCAG success criterion 1.3.5: Identify input purpose](https://www.w3.org/WAI/WCAG22/Understanding/identify-input-purpose.html), for instance `"bday-day"`. See the [Autofill section in the HTML standard](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) section in the HTML standard for full list of attributes that can be used.'
```

Are these single quotes accidentally in here?

```suggestion
        description: Attribute to meet [WCAG success criterion 1.3.5: Identify input purpose](https://www.w3.org/WAI/WCAG22/Understanding/identify-input-purpose.html), for instance `"bday-day"`. See the [Autofill section in the HTML standard](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) section in the HTML standard for full list of attributes that can be used.
```

---

No these are intentional. The colon in the text itself (after '1.3.5') breaks yaml formatting and the quote tells yaml to treat this like a string again. Normally we don't do this since single lines of text without any of yaml's special characters (eg: colons) are automatically treated like standard strings of text. It's a pain that this breaks convention but I think it's ok as an exception. If anyone tries to remove it, they'll realise very quickly why it's there.